doc/_admin-guide/110_Template_and_rewrite/000_Customize_message_format/004_Macros_of_syslog-ng.md

```diff
@@ -430,6 +430,43 @@ database column.
 
 An alias of the ${TZOFFSET} macro.
 
+## ${TRANSPORT}
+
+This macro is available in syslog-ng OSE 4.5 and later versions. The syslog-ng OSE application automatically populates this name-value pair with the transport-mechanism used to retrieve the message. The value depends on the source driver that received the message. The implemented values are the following:
+
+* BSD syslog drivers `tcp()`, `udp()` & `network()`
+  * RFC-3164 + tls
+  * RFC-3164 + tcp
+  * RFC-3164 + udp
+  * RFC-3164 + proxied-tls
+  * RFC-3164 + \<custom logproto like altp\>
+
+* UNIX domain drivers `unix-dgram()`, `unix-stream()`
+  * `unix-stream`
+  * `unix-dgram`
+
+* RFC-5424 style syslog `syslog()`:
+  * RFC-5426: syslog over udp
+  * RFC-5425: syslog over tls
+  * RFC-6587: syslog over tcp
+  * RFC-5424 + \<custom logproto like altp\>: syslog over a logproto plugin
+
```

Comment:
Missing cases and strings are

`rfc6587+proxied-tcp`
`rfc5424+proxied-tls`
`rfc5424+proxied-tls-passthrough"`